src/java/edu/ucsd/library/dams/util/EncryptedStreamNameModule.java

```diff
@@ -287,24 +283,21 @@ else if (extension.equals("flv"))
 		try
 		{
 			// pairpath based on objid
-			for( int i = 0; i < (objid.length() - 1); i += 2 )
+			for( int i = 0; i < (binaryname.length() - 1) && i < 5; i += 2 )
```

Comment:
What does 5 signify here? For someone coming into this code without context, it might be hard to understand. Is there a name we could give to 5 in this context?